Package.swift

```diff
@@ -0,0 +1,9 @@
+import PackageDescription
+
+let package = Package(
+    name: "GRDB.swift",
+    dependencies: [
+        .Package(url: "https://github.com/groue/SQLiteSDK.git", majorVersion: 0)
```

Comment:
Yes, I used SQLiteSDK because there were 'import SQLiteSDK' in most of the files and this allowed not patching source code at all. I had to restore these imports as they were removed in latest master. It's probably better to rename the repo to "CSQLite" and adjust imports accordingly.